3.2/Dockerfile

```diff
@@ -1,28 +1,29 @@
+# MongoDB image for OpenShift.
```

Comment:
@bparees It is still true? Does PM still reference these images as OpenShift or RHSCL?